cli/bin/env-setup

```diff
@@ -6,12 +6,8 @@ else
     BIN_DIR=$PWD/bin
 fi
 
+# real, absolute path to BIN_DIR
 FULL_BIN_PATH=$(python -c "import os; print(os.path.realpath('$BIN_DIR'))")
 
+# ensure BIN_DIR is prepended to PATH
 expr "$PATH" : "${FULL_BIN_PATH}.*" > /dev/null || export PATH=$FULL_BIN_PATH:$PATH
-export DCOS_CONFIG=~/.dcos/dcos.toml
-
-if [ ! -f "$DCOS_CONFIG" ]; then
```

Comment:
As you see in this PR, we create the file if `dcos` tries to access it and sees it doesn't exist.  So the install script does not create the file directly.

So we shouldn't need this creation block anywhere.

And `env-setup` is still necessary to update the $PATH, so I don't think we can remove the `source` line you referenced.